### PR TITLE
Update js-array-methods.json

### PR DESCRIPTION
**Pull request by mdmahinpro**
new method added

### CODE REVIEW

File: data/js-array-methods.json [hunk 1]
```diff
@@ -30,6 +30,10 @@
                     "definition": "অ্যারের উপাদানগুলোকে অ্যারের অন্য অবস্থানে কপি করে",
                     "code": "arr.copyWithin(1, 3)"
                 },
+                {
+                    "definition": "উপাদান দিয়ে নতুন অ্যারে তৈরি",
+                    "code": "arr.of("A", "B", "C")"
```
Comment (sabbirshawon) [inline data/js-array-methods.json]: fix this line

**Review by sabbirshawon — CHANGES_REQUESTED** [review]
fix the problem and send pr again... dont use " " within " "

**Comment by fahadali32** [comment]
"arr.of('A', 'B', 'C')"
use this

**Review by mdmahinpro — COMMENTED** [review]
error fixed

**Review by shaonkabir8 — APPROVED** [review]

**Pull request merged**